pkcs11/pkcs11_rsa.c

```diff
@@ -320,19 +320,24 @@ static int rsa_sign_verify_pss(int devId)
     return ret;
 }
 #endif /* ifdef WC_RSA_PSS */
-#endif /* ifndef NO_RSA */
+
+/* Define maximum RSA key size in bits */
+#define MAX_RSA_KEY_BITS 2048
+
+#ifndef NO_RSA
```

Comment:
This line needs removed. It doesn't build because `#ifndef NO_RSA` is already defined above. Please remove this line only.

```
gcc -o pkcs11_rsa pkcs11_rsa.c -Wall -I/usr/local/include -Os -L/usr/local/lib -lm -lwolfssl
pkcs11_rsa.c:30:2: error: unterminated conditional directive
   30 | #ifndef NO_RSA
      |  ^
1 error generated.
```